### ISSUE TITLE
Room only joined after first message

### ISSUE THREAD

**Issue by pcworld** [mautrix/signal#411]
I got an invite to a Signal room. As the bridge does not yet support relaying room invites, I joined the room via signal-cli.
However, the chat room did not appear in the Matrix bridge until someone wrote a message in that room.
I would expect that the room appears as soon as I have joined.

**Comment by maltee1** [comment]
That's not an actual bug, it's a missing feature and it's known: https://github.com/mautrix/signal/blob/main/ROADMAP.md

I suppose you can expect that to be added eventually.

**Comment by pcworld** [comment]
Ok. I did read the Roadmap, but I'm not sure which of the roadmap items applies to this. Is this under "Automatic portal creation"? Currently, this has the subitems "When receiving message" (checked), and "After login" (unchecked). However, my issue was neither case, I suppose this case would be called "After accepting invite to room".

**Comment by maltee1** [comment]
Fair point. I suppose that roadmap item is inaccurate/incomplete. However, the bridge is under heavy development and lots of features are yet to be added. I'm sure it's on the radar.

**Comment by maltee1** [comment]
This should be fixed by now, partially by v2 and partially by #528
